src/main/res/values/strings.xml

```diff
@@ -1451,6 +1451,8 @@
     <string name="card_name_strelka">Strelka</string>
     <string name="strelka_long_serial">Strelka long number</string>
     <string name="card_name_troika_strelka_hybrid">Troika+Strelka</string>
+    <!-- Translators: Druzhinnik is a member of Russian Neighbourhood Watch -->
+    <string name="troika_druzhinnik_card">Druzhinnik&quot;s card</string>
```

Comment:
It looks like this is used in the context of a Subscription name.

I think using the non-possessive form ("Druzhinnik card") is more consistent with the other cards (eg: "school card", "student card", etc.)

Also, is this a special Troika card for Druzhinniks, or is this a concession that can be added to any Troika?

Reply:
It's a special card different from normal Troika, both in printed design and layout (they don't have balance sector)